apis/clusters/v1beta1/opensearch_types.go

```diff
@@ -56,6 +56,8 @@ type OpenSearchSpec struct {
 	UserRefs                 []*UserReference        `json:"userRefs,omitempty"`
 	//+kubuilder:validation:MaxItems:=1
 	ResizeSettings []*ResizeSettings `json:"resizeSettings,omitempty"`
+	//+kubuilder:validation:MaxItems:=1
+	OpenSearchIngestNodes []*OpenSearchIngestNodes `json:"ingestNodes,omitempty"`
```

Comment:
OpenSearch- prefix in field name is unnecessary.

Reply:
Fixed